TADashboard/src/main/java/page_objects/DashboardPage.java

```diff
@@ -1,11 +1,19 @@
 package page_objects;
 
 import com.logigear.control.common.imp.Label;
+import com.logigear.control.common.imp.Link;
 
 public class DashboardPage {
     private Label lblWelcome = new Label("css=a[href='#Welcome']");
+    private Link lnkDataProfiles = new Link("css=li>ul>li>a[href='profiles.jsp']");
+    private Link lnkAdminister = new Link("css=a[href='#Administer']");
 
     public boolean doesWelcomeLabelDisplay() {
         return lblWelcome.isVisible();
     }
-}
+
+    public void selectAdministerMenu(DataProfilesPage dataProfilesPage){
```

Comment:
what is this? what is DataProfilesPage, and where you are using it in this method?